python/lsst/dax/apdb/apdbSql.py

```diff
@@ -160,15 +160,15 @@ class ApdbSqlConfig(ApdbConfig):
     htm_index_column = Field[str](
         default="pixelId", doc="Name of a HTM index column for DiaObject and DiaSource tables"
     )
-    ra_dec_columns = ListField[str](default=["ra", "decl"], doc="Names ra/dec columns in DiaObject table")
+    ra_dec_columns = ListField[str](default=["ra", "decl"], doc="Names of ra/dec columns in DiaObject table")
     dia_object_columns = ListField[str](
         doc="List of columns to read from DiaObject, by default read all columns", default=[]
     )
     prefix = Field[str](doc="Prefix to add to table names and index names", default="")
     namespace = Field[str](
         doc=(
-            "Namespace or schema name for all tables in APDB database. "
-            "Presently only makes sense for PostgresQL backend. "
+            "Schema name for all tables in APDB database. "
+            "Presently only recommended for PostgreSQL backend, should not be set for SQLite. "
```

Comment:
I think the "should not be set" is still misleading -- after all, nothing bad happens if you do set it! If you feel SQLite needs to be mentioned specifically, maybe "is redundant with SQLite file name"?

Reply:
Also pinging @BrunoSanchez for a second opinion.

Reply:
The connection between database file name and schema name is not as straightforward for SQLite. Default schema name is `main` and it is not related to a file name. Schema name for a database can be specified in `ATTACH` statement when connecting other files, but we never use `ATTACH` with APDB. I think the only reasonable value for `namespace` in SQLite case is either `None` or `"main"`, and it does not make much sense to specify "main", so I think it's better to say that it should not be set.

Reply:
I think that if you specify that the namespace keyword is not intended for use with SQLite back-end, and unexpected behavior might result then would be enough. Still, I don't think it is a good idea to put long explanations in the docstring.

Reply:
Indeed, I was wrong and `namespace` does *not* work with SQLite (it fails with `make_apdb.py`). However, this invalidates the original Jira issue, so I suggest either leaving this entry unchanged or saying "Presently only works for PostgreSQL backend."

Reply:
OK, I changed it to "Presently only works ..."